packages/grafana-ui/src/components/Forms/FieldArray.story.tsx

```diff
@@ -13,10 +14,17 @@ export default {
     docs: {
       page: mdx,
     },
+    controls: {
+      exclude: ['name', 'keyName', 'control', 'shouldUnregister'],
+    },
   },
-};
+  argTypes: {
+    ContainerWidth: { control: { type: 'range', min: 100, max: 500, step: 10 } },
+    ContainerHeight: { control: { type: 'range', min: 100, max: 500, step: 10 } },
+  },
+} as Meta;
 
-export const simple = () => {
+export const Simple: Story = (args) => {
```

Comment:
Seems like args is not used in tis story?

Reply:
yes...but we want to give the user the ability to resize the story container thus we had to configure the story to use controls.

Reply:
What I mean is that you can do `export const Simple: Story = () => {`because the `args` param is unused.

Reply:
We make this possible by passing some story args to the decorators via the story context

Reply:
True, but the `args` param is not used anywhere inside this story or am I missing smth here?